frontend/src/pages/Dashboard.jsx

```diff
@@ -21,10 +22,16 @@ const styles = {
 
 class Dashboard extends Component {
 
+    errorMessage(message) {
```

Comment:
we could abstract this out into it's own function

Reply:
or component

Reply:
Isnt this already its own function?

Reply:
todo for #234